guide/01-getting-started/install-and-set-up.ipynb

```diff
@@ -4,234 +4,100 @@
    "cell_type": "markdown",
```

Comment:
<p>"using the steps below"... but there are no steps below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/764/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>